stable/grafana/README.md

```diff
@@ -90,7 +90,7 @@ kind: ConfigMap
 metadata:
   name: sample-grafana-dashboard
   labels:
-     grafana_dashboard: 1
+    {{ .Release.Name }}-grafana-dashboard: 1
```

Comment:
Should this use the helper too? For consistency?

Reply:
I think it will be better as is. The reason being that, depending on the way your charts are organized, the helper may or may not work. In one of the previous resolved comments, there are more detailed examples.